src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.NonAndroid.cs

```diff
@@ -134,7 +134,7 @@ private static List<string> ParseTimeZoneIds(StreamReader reader)
             {
                 if (!string.IsNullOrEmpty(zoneTabFileLine) && zoneTabFileLine[0] != '#')
                 {
-                    // the format of the line is "country-code \t coordinates \t TimeZone Id \t comments"
+                    // the format of the line is "ISO 3166 territory code \t coordinates \t TimeZone Id \t comments"
```

Comment:
It is interesting that the standard is using `Country Code` https://www.iso.org/iso-3166-country-codes.html. I don't mind using `territory` though.